<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelPointSet</class>
 <widget class="QScrollArea" name="PanelPointSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScrollArea</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="widgetResizable">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="scrollAreaWidgetContents">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>314</width>
     <height>816</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutToolbar">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolBar" name="toolbar">
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="floatable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="LayerTreeWidget" name="treeWidgetLayers">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>145</height>
         </size>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelFileName">
        <property name="text">
         <string>File name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditFileName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelOpacity">
        <property name="text">
         <string>Opacity</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSlider" name="sliderOpacity">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBoxOpacity">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="labelSplineColor">
        <property name="text">
         <string>Spline color</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="comboBoxSplineColor">
        <item>
         <property name="text">
          <string>Solid Color</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Heatscale</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="labelScalarMap">
        <property name="text">
         <string>Scalar map</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QComboBox" name="comboBoxScalarMap"/>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="labelMin">
        <property name="text">
         <string>Min</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QSlider" name="sliderMin">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditMin">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="labelMid">
        <property name="text">
         <string>Mid</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QSlider" name="sliderMid">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditMid">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="labelMax">
        <property name="text">
         <string>Max</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QSlider" name="sliderMax">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditMax">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="labelOffset">
        <property name="text">
         <string>Offset</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QSlider" name="sliderOffset">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditOffset">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="labelSplineRadius">
        <property name="text">
         <string>Spline radius</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <widget class="QLineEdit" name="lineEditSplineRadius">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelRadius">
        <property name="text">
         <string>Radius</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineEditRadius">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="checkBoxSnapToCenter">
        <property name="text">
         <string>Snap to voxel center</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="checkBoxShowSpline">
        <property name="text">
         <string>Show spline</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QtColorPicker" name="colorpickerPointColor" native="true"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelColor_2">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QtColorPicker" name="colorpickerSplineColor" native="true"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtColorPicker</class>
   <extends>QWidget</extends>
   <header>qtcolorpicker.h</header>
   <container>1</container>
   <slots>
    <signal>colorChanged(QColor)</signal>
    <signal>currentColorChanged(QColor)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>LayerTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>LayerTreeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>sliderOpacity</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PanelPointSet</receiver>
   <slot>OnSliderOpacity(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>-22</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditRadius</sender>
   <signal>textEdited(QString)</signal>
   <receiver>PanelPointSet</receiver>
   <slot>OnLineEditRadius(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PanelPointSet</receiver>
   <slot>OnSliderMin(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>644</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderMid</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PanelPointSet</receiver>
   <slot>OnSliderMid(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>605</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PanelPointSet</receiver>
   <slot>OnSliderMax(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>668</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderOffset</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PanelPointSet</receiver>
   <slot>OnSliderOffset(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>715</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditSplineRadius</sender>
   <signal>textEdited(QString)</signal>
   <receiver>PanelPointSet</receiver>
   <slot>OnLineEditSplineRadius(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>754</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditMin</sender>
   <signal>textEdited(QString)</signal>
   <receiver>PanelPointSet</receiver>
   <slot>OnLineEditMin(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>624</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditMid</sender>
   <signal>textEdited(QString)</signal>
   <receiver>PanelPointSet</receiver>
   <slot>OnLineEditMid(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>699</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditMax</sender>
   <signal>textEdited(QString)</signal>
   <receiver>PanelPointSet</receiver>
   <slot>OnLineEditMax(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>498</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>738</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditOffset</sender>
   <signal>textEdited(QString)</signal>
   <receiver>PanelPointSet</receiver>
   <slot>OnLineEditOffset(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>527</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>766</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxScalarMap</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PanelPointSet</receiver>
   <slot>OnComboScalarMap(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnSliderOpacity(int)</slot>
  <slot>OnSliderMin(int)</slot>
  <slot>OnSliderMid(int)</slot>
  <slot>OnSliderMax(int)</slot>
  <slot>OnSliderOffset(int)</slot>
  <slot>OnLineEditMin(QString)</slot>
  <slot>OnLineEditMid(QString)</slot>
  <slot>OnLineEditMax(QString)</slot>
  <slot>OnLineEditOffset(QString)</slot>
  <slot>OnLineEditRadius(QString)</slot>
  <slot>OnLineEditSplineRadius(QString)</slot>
  <slot>OnComboScalarMap(int)</slot>
 </slots>
</ui>
